src/test/java/com/accelad/acctive/sim/kernel/core/evaluator/diff/BetweenStrictlyLessThanMaxFunctionTest.java

```diff
@@ -18,9 +18,9 @@
 import com.accelad.math.nilgiri.autodiff.DifferentialFunction;
 import com.accelad.math.nilgiri.autodiff.Variable;
 
-public class BetweenFunctionTest {
+public class BetweenStrictlyLessThanMaxFunctionTest {
```

Comment:
It seems that there are no test for behaviour when value lower that lower bound.